### ISSUE TITLE
FR 3.7.1 Declare winner if other player(s) forfeits

### ISSUE THREAD

**Issue by Trulsmatias** [Jonas-C/TDT4240-Gaalf#70]
FR 3.7.1: If  a  player  forfeits  or  exits  a  game  the  other  player should  be  declared  as  winner  and  the  game  should end.

Right now the player left in the game just keeps playing

**Comment by Trulsmatias** [comment]
Relevant to issue #63

**Comment by Jonas-C** [comment]
> FR 3.7.1: If a player forfeits or exits a game the other player should be declared as winner and the game should end.

We can probably just leave this as-is. It's not really a big deal if the remaining player can continue playing. If they're tearing through a course I doubt they'd want the game to be cancelled.

**Comment by Trulsmatias** [comment]
Yeah, I do think I agree with that. There score is shown in the end regardless
